src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets

```diff
@@ -42,10 +42,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DotnetCliToolTargetFramework>netcoreapp2.2</DotnetCliToolTargetFramework>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <IncludeBuildOutput Condition=" '$(PackAsTool)' == 'true' ">false</IncludeBuildOutput>
-    <PackageType Condition=" '$(PackAsTool)' == 'true' ">DotnetTool</PackageType>
-    <RuntimeIdentifiers Condition=" '$(PackAsTool)' == 'true' ">$(RuntimeIdentifiers);$(PackAsToolShimRuntimeIdentifiers)</RuntimeIdentifiers>
+  <PropertyGroup Condition=" '$(PackAsTool)' == 'true' ">
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <PackageType>DotnetTool</PackageType>
+    <RuntimeIdentifiers>$(RuntimeIdentifiers);$(PackAsToolShimRuntimeIdentifiers)</RuntimeIdentifiers>
+    <RollForward Condition="'$(RollForward)' == ''">Major</RollForward>
```

Comment:
@baronfel Is it possible for us to add some logic to detect if we did do a roll-forward and then errored.. and if so warn the user about the roll forward and tell them to use an older SDK?

Reply:
This is not a bad idea. What happens with tools is that 
* we eventually write a `DotnetToolSettings.xml` file that looks like this:

```
<?xml version="1.0" encoding="utf-8"?>
<DotNetCliTool Version="1">
  <Commands>
    <Command Name="fsautocomplete" EntryPoint="fsautocomplete.dll" Runner="dotnet" />
  </Commands>
</DotNetCliTool>
```

* the rollforward is set in the runtimeconfig.json to whatever we infer here

We probably need to set some other kind of data in the `DotnetToolSettings.xml` to signal that _we_ opted the user into auto-rollforward so we could give an error here.

Reply:
I don't think we need to do this (add the data to do error detection) for this PR - I think this behavior is correct and good and what we should have done all along, and a breaking change notice is enough. Virtually every tool author I've spoken to has wondered why this wasn't the default, so I don't think there's a user experience gap that we need to cover here.

If we get feedback from users post-preview5 that this was confusing, we can certainly come back to this though!